doc/managing-wallet.md

```diff
@@ -0,0 +1,85 @@
+# Managing the Wallet
+
+## 1. Backing Up and Restoring The Wallet
+
+### 1.1 Creating the Wallet
+
+Since version 0.21, Bitcoin Core no longer has a default wallet.
+Wallets can be created with the RPC command `createwallet`. More information about this command can be found running `bitcoin-cli help createwallet`.
+
+The following command, for example, creates a descriptor wallet:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true`
+
+The `descriptors` parameter can be omitted if the intention is to create a legacy wallet.
+
+For now, the default type is the legacy wallet, but that should change in the near future.
+
+Wallets are created in the `wallets` folder of the data directory. The default folder varies by operating systems, as shown below. The user can also change the default by using the `-datadir` or `-walletdir` initialization parameters.
+
+| Operating System | Default wallet directory                                    |
+| -----------------|:------------------------------------------------------------|
+| Linux            | `home/<user>/.bitcoin/wallets`                              |
+| Windows          | `C:\Users\<user>\AppData\Roaming\Bitcoin\wallets`           |
+| macOS            | `/Users/<user>/Library/Application Support/Bitcoin/wallets` |
+
+### 1.2 Encrypting the Wallet
+
+The `wallet.dat` file is not encrypted by default and is, therefore, vulnerable if an attacker gains access to the device where the wallet or the backups are stored.
+
+The wallet may be encrypted with the following command:
+
+`$ bitcoin-cli -rpcwallet="wallet-01" encryptwallet "passphrase"`
+
+The `encryptwallet` command is used only when the wallet has not been encrypted yet. Otherwise, the `walletpassphrasechange` command should be used.
+
+`$ bitcoin-cli -rpcwallet="wallet-01" walletpassphrasechange "oldpassphrase" "newpassphrase"`
+
+The argument passed to `-rpcwallet` is the name of the wallet to be encrypted.
+
+The term "encrypt the wallet" used here is not very accurate. This command encrypts only the private key. All other wallet information, such as transactions, is still visible.
+
+The wallet can also be encrypted in the `createwallet` command via the `passphrase` argument:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true passphrase="passphrase"`
+
+Note that if the passphrase is lost, all the coins in the wallet will also be lost forever.
+
+### 1.3 Backing Up the Wallet
+
+Wallets can be safely copied to another destination. This backup file should be stored on offline devices only, such as on a USB drive or an external hard drive.
```

Comment:
Not sure about this. Offline device is better than Online device but still not safe.

Reply:
If not offline or online, where?

Reply:
I think he meant that not _any_ offline device is safe. But I would say the lack of security @prayank23 pointed out on some offline devices may be out of the scope of this documentation. 

What I understand of `Wallets can be safely copied` is it refers to the fact that the funds will be recoverable just backuing up that file. Then he is emphasizing the fact that online storage is a bad idea, which I think we all agree.

Reply:
> If not offline or online, where?

Offline is okay but USB drives cannot be trusted and in some cases external hard drive as well. So if I had to rephrase the above thing and add few more options, it would be something like this:

_Wallets should be safely copied to another destination. This backup file can be stored on multiple offline devices that you trust will work in emergency, never connected to online devices and no malware. Test your backups regularly to avoid any issues in future. Also printing the output for `dumpwallet` on a sheet of paper could be helpful in some cases._

> I think he meant that not any offline device is safe. But I would say the lack of security @prayank23 pointed out on some offline devices may be out of the scope of this documentation. 

Agree. Can't go in to details but this repository is used by lot of users with different levels of technical expertise so nobody should trust one file on some USB.

Reply:
Sure, there are security issues with USB devices (propriety firmware, etc) - but I think it is out of scope for this doc (for example, we don't tell our users to only use Linux/Power ISA devices).

I do like the idea of including `dumpwallet` as an option - especially with [this](https://github.com/bitcoin/bitcoin/pull/11803) PR hopefully being merged (although we should inform that this option would lose metadata)

Reply:
The document originally had two sections on `dumpwallet` and `importwallet`. But it was removed for the reason cited in https://github.com/bitcoin/bitcoin/pull/22523#discussion_r674488255.
From the reviews, it seems that there is a preference for a shorter and less verbose document.
But I agree that `dumpwallet` backup and security notes are a good follow-up.

Reply:
Makes sense.  Maybe a `listdescriptors` follow up too https://github.com/bitcoin/bitcoin/pull/21500

Reply:
Added the suggested security considerations.